bindings/protocols/mqtt/index.template.html

```diff
@@ -173,9 +188,215 @@ <h3>Possible mappings</h3>
   </section>
     <section>
       <h2>Examples</h2>
-      <p>The following examples show how to use the MQTT protocol in a Thing Description.</p>
-      <p class="ednote">TODO:</p>
+      <p>
+        The following examples show how to use the MQTT protocol in a Thing Description. The [[[#example-simple-event]]] 
+        shows a simple form that indicates to compliant clients to subscribe or unsubscribe to the topic <code>thing1/events/overheating</code>. 
+      </p>
+      <pre id="example-simple-event" class="example" title="Read a single coil">
+        {
+            "href": "mqtt://broker.com:1883",
+            "op": [
+                "subscribeevent",
+                "unsubscribeevent"
+            ],
+            "mqv:filter": "thing1/events/overheating" 
+        }
+      </pre>
+      <p>
+        [[[#example-simple-publish]]] shows how to use the <code>mqv:publish</code> packet when invoking a 
+        command. Remember that the <code>invokeaction</code> operation is mapped to <code>mqv:publish</code>
+        control packet in [[[#default-mappings]]]. 
+      </p>
+      <pre id="example-simple-publish" class="example" title="Read a single coil">
+        {
+            "href": "mqtt://broker.com:1883",
+            "op": [
+                "invokeaction",
+            ], 
+            "mqv:topic": "application/devices/thing1/program/commands/reset",
+        }
+      </pre>
+      <p>
+        Continuing with increasingly more complex examples, [[[#example-property]]] shows how to use the MQTT protocol when handling properties.
+        <code>mqv:retain</code> is used to indicate the ability to read the property rather than just observe it. 
+      </p>
+      <pre id="example-property" class="example" title="Read a single coil">
+        {
+            "href": "mqtt://broker.com:1883",
+            "op": [
+                "readproperty",
+                "writeproperty",
+                "observeproperty"
+            ], 
+            "mqv:retain" : true,
+            "mqv:topic": "application/devices/thing1/properties/test",
+            "mqv:filter": "application/devices/thing1/properties/test",
+        }
+      </pre>
+      <p>
+        If the implementer want to give the ability to WoT clients to conveniently observe all properties in a single call, it is possible
+        to use observerallproperties form in the root of a Thing Description. 
```

Comment:
I think it would be nice to mention that this means all messages published to the topics under `application/devices/thing1/properties/` will be delivered.